spec/locales-currencies-tz.html

```diff
@@ -164,59 +164,228 @@ <h1>
+        </dd>
+        <dt>redefinition</dt>
+        <dd>true</dd>
       </dl>
+      <p>This definition supersedes the definition provided in <emu-xref href="#sec-availablenamedtimezoneidentifiers"></emu-xref>.</p>
       <emu-alg>
-        1. If one of the Zone or Link names of the IANA Time Zone Database is an ASCII-case-insensitive match for _timeZone_, return *true*.
-        1. If _timeZone_ is an ASCII-case-insensitive match for *"UTC"*, return *true*.
-        1. Return *false*.
+        1. Let _identifiers_ be a List containing the String value of each Zone or Link name in the IANA Time Zone Database.
+        1. Assert: No element of _identifiers_ is an ASCII-case-insensitive match for any other element.
+        1. Assert: Every element of _identifiers_ identifies a Zone or Link name in the IANA Time Zone Database.
+        1. Sort _identifiers_ into the same order as if an Array of the same values had been sorted using %Array.prototype.sort% with *undefined* as the argument.
+        1. Let _result_ be a new empty List.
+        1. For each element _identifier_ of _identifiers_, do
+          1. Let _primary_ be _identifier_.
+          1. If _identifier_ is listed in the first column of <emu-not-ref>Legacy</emu-not-ref> POSIX Time Zone Identifier Mapping <emu-xref href="#table-posix-time-zone-identifier-mapping"></emu-xref>, then
+            1. Set _primary_ to the value in the second column of the row where the first column is _identifier_.
+          1. Else,
+            1. NOTE: The algorithm steps below are intended to mimic the behaviour of the *icu::TimeZone::getIanaID()* API of the International Components for Unicode (ICU) <a href="https://icu.unicode.org/">https://icu.unicode.org/</a>.
+               The steps in this section are provided for testing conformance of ICU-based ECMAScript implementations, and to ensure compatibility between ICU-based and non-ICU-based implementations.
+            1. If _identifier_ is present in the *TZ* column of *zone.tab* of the IANA Time Zone Database, then
+              1. Set _primary_ to _identifier_.
+            1. Else if _identifier_ is a Link in the IANA Time Zone Database, then
+              1. Let _zone_ be the Zone name that _identifier_ resolves to, according to the rules for resolving Link names in the IANA Time Zone Database.
+              1. If _zone_ starts with *"Etc/"*, then
+                1. Set _primary_ to _zone_.
+              1. Else,
+                1. Let _identifierCountyCode_ be the ISO 3166-2 country code <a href="https://www.iso.org/iso-3166-country-codes.html">https://www.iso.org/iso-3166-country-codes.html</a> whose territory contains the geographical area corresponding to _identifier_.
+                1. Let _zoneCountyCode_ be the ISO 3166-2 country code whose territory contains the geographical area corresponding to _zone_.
+                1. If _identifierCountyCode_ is _zoneCountyCode_, then
```

Comment:
```suggestion
                1. Let _identifierCountryCode_ be the ISO 3166-2 country code <a href="https://www.iso.org/iso-3166-country-codes.html">https://www.iso.org/iso-3166-country-codes.html</a> whose territory contains the geographical area corresponding to _identifier_.
                1. Let _zoneCountryCode_ be the ISO 3166-2 country code whose territory contains the geographical area corresponding to _zone_.
                1. If _identifierCountryCode_ is _zoneCountryCode_, then
```

Reply:
Lol, I had to look for several minutes before I could even spot the change. No wonder I missed the typo the first time. Thanks for catching this, fixed now.